src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/VirtualMachineScaleSetTests.ps1

```diff
@@ -202,11 +208,23 @@ function Test-VirtualMachineScaleSet
         $st = Start-AzureVMSS -ResourceGroupName $rgname -VMScaleSetName $vmss.Name;
         $st = Restart-AzureVMSS -ResourceGroupName $rgname -VMScaleSetName $vmss.Name;
 
-        $instanceListParam = New-AzureComputeParameterObject -FriendlyName VirtualMachineScaleSetVMInstanceIDs;
+        $instanceListParam = @();
         for ($i = 0; $i -lt $vmList.Count; $i++)
         {
-            $instanceListParam.InstanceIDs.Add($i);
+            $instanceListParam += $i.ToString();
         }
+
+        $argList = New-AzureComputeArgumentList -MethodName VirtualMachineScaleSetPowerOffInstances;
+        $argList[0].Value = $rgname;
+        $argList[1].Value = $vmss.Name;
+        $argList[2].Value = $instanceListParam;
+        $args = @()
+        for ($i = 0; $i -lt $argList.Length; $i++)
+        {
+            $args += , $argList[$i].Value;
+        }
+
+        #$vmssResult = Invoke-AzureComputeMethod -MethodName VirtualMachineScaleSetPowerOffInstances -ArgumentList $args;
```

Comment:
I'm testing both Invoke-AzureComputeMethod and the cmdlets itself.  I don't have the subscription for VMSS, so I tested both using playback mode, and that is why I commented out Invoke-AzureComputeMethod part after I testing it.